code/modules/uplink/uplink_item.dm

```diff
@@ -1079,7 +1085,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 			against other attacks. Pair with an Energy Sword for a killer combination."
 	item = /obj/item/shield/energy
 	cost = 16
-	surplus = 20
+	surplus = 0 //considering I am not absolutly certain if game mode exclusion affects this var better set it to 0
```

Comment:
It does apply. Eshields won't be possible to purchase outside of nukie or gang rounds.